billing_group.go

```diff
@@ -0,0 +1,118 @@
+package aiven
+
+type (
+	// BillingGroup represents an billing group
+	BillingGroup struct {
+		BillingGroupRequest
+		Id string `json:"billing_group_id"`
+	}
+
+	// BillingGroupRequest  is the request from Aiven for the billing group endpoints.
+	BillingGroupRequest struct {
+		BillingGroupName string          `json:"billing_group_name,omitempty"`
+		AccountId        *string         `json:"account_id,omitempty"`
+		CardId           *string         `json:"card_id,omitempty"`
+		VatId            *string         `json:"vat_id,omitempty"`
+		BillingCurrency  *string         `json:"billing_currency,omitempty"`
+		BillingExtraText *string         `json:"billing_extra_text,omitempty"`
+		BillingEmails    []*ContactEmail `json:"billing_emails,omitempty"`
+		Company          *string         `json:"company,omitempty"`
+		AddressLines     []string        `json:"address_lines,omitempty"`
```

Comment:
I think these can no longer be set in billing group requests (they are returned in responses, though)